src/reducers/reducerCart.js

```diff
@@ -0,0 +1,37 @@
+import { INCREMENT_QUANTITY, DECREMENT_QUANTITY, REMOVE_ITEM_FROM_CART } from '../constants/actionTypes';
+
+const initialState = {
+  userCart: [{
+    id: 10, quantity: 1, color: 'white', size: 'XS'
+  },
+  {
+    id: 12, quantity: 1, color: 'red', size: 'S'
+  }
+  ]
+};
+
+// initial state is not left empty here just to show at least anything since store is not connected yet//
```

Comment:
🐻

Reply:
Don't forget to remove clean-up this later

Reply:
yeap!